modules/inputSpinner/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@diana-ui/inputspinner",
+  "version": "0.0.1",
+  "main": "lib/index.js",
+  "module": "module/index.js",
+  "files": [
+    "lib",
+    "module"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "build": "yarn compile:lib && yarn compile:module",
+    "compile:lib": "tsc --target es5 --module es2015 --outDir lib",
+    "compile:module": "tsc --target es6 --module esnext --outDir module",
+    "clean:files": "rimraf lib module",
+    "prepare": "yarn clean:files && yarn build",
+    "watch": "npm-watch"
+  },
+  "dependencies": {
+    "@diana-ui/base": "0.1.2",
+    "@diana-ui/types": "0.1.2",
+    "@diana-ui/typography": "0.1.2",
+    "react": ""
+  },
```

this is already fixed, but not merged yet . can you remove this from here, and add
```suggestion
  "dependencies": {
    "@diana-ui/base": "0.1.2",
    "@diana-ui/types": "0.1.2",
    "@diana-ui/typography": "0.1.2",
  },
  "peerDependencies": {
        "react": "^16.12.0"
  }
```